src/onnx/parse_multinomial.cpp

```diff
@@ -46,38 +46,79 @@ struct parse_multinomial : op_parser<parse_multinomial>
             dtype = info.attributes.at("dtype").i();
         shape::type_t output_type = get_type(dtype);
 
+        size_t batch_size = 1;
+        if(contains(info.attributes, "batch_size"))
+            batch_size = info.attributes.at("batch_size").i();
```

Comment:
What is this attribute for? It's not in the ONNX spec

Reply:
That's right, removing.  batch_size appears only implicitly.